src/react/components/pages/editorPage/editorPage.test.tsx

```diff
@@ -61,86 +61,6 @@ describe("Editor Page Component", () => {
         expect(loadProjectSpy).not.toBeCalled();
         expect(editorPage.prop("project")).toEqual(testProject);
     });
-
-    it("Loads project assets when state changes", (done) => {
```

Comment:
These tests shouldn't be removed either... Phil is adding new tests, but these are all still valid